src/main/java/ToDos.java

```diff
@@ -0,0 +1,23 @@
+public class ToDos extends Task{
+    public ToDos(String itemName) {
+        super(itemName);
+    }
+
+    @Override
+    public void printTask() {
+        String status = new String();
```

Comment:
You could consider having a getStatusIcon method in your task.java and simplify your printTask code for Todos Deadline and Events code